customize_fusion_values.sh

```diff
@@ -0,0 +1,153 @@
+}
+
+SCRIPT_CMD="$0"
+MY_VALUES="$1"
+
+if [ "$MY_VALUES" == "" ]; then
+  print_usage "$SCRIPT_CMD" "Please provide the name of the values yaml file to create as the first arg to this script!"
+  exit 1
+fi
+
+# start parsing
+shift 1
+
+if [ $# -gt 1 ]; then
+  while true; do
+    case "$1" in
+        -c)
+            if [[ -z "$2" || "${2:0:1}" == "-" ]]; then
+              print_usage "$SCRIPT_CMD" "Missing value for the -c parameter!"
+              exit 1
+            fi
+            CLUSTER_NAME="$2"
+            shift 2
+        ;;
+        -r)
+            if [[ -z "$2" || "${2:0:1}" == "-" ]]; then
+              print_usage "$SCRIPT_CMD" "Missing value for the -r parameter!"
+              exit 1
+            fi
+            RELEASE="$2"
+            shift 2
+        ;;
+        --provider)
+            if [[ -z "$2" || "${2:0:1}" == "-" ]]; then
+              print_usage "$SCRIPT_CMD" "Missing value for the --provider parameter!"
+              exit 1
+            fi
+            PROVIDER="$2"
+            shift 2
+        ;;
+        --prometheus)
+            if [[ -z "$2" || "${2:0:1}" == "-" ]]; then
+              print_usage "$SCRIPT_CMD" "Missing value for the --prometheus parameter!"
+              exit 1
+            fi
+            PROMETHEUS_ON="$2"
+            shift 2
+        ;;
+        --num-solr)
+            if [[ -z "$2" || "${2:0:1}" == "-" ]]; then
+              print_usage "$SCRIPT_CMD" "Missing value for the --num-solr parameter!"
+              exit 1
+            fi
+            SOLR_REPLICAS=$2
+            shift 2
+        ;;
+        --solr-disk-gb)
+            if [[ -z "$2" || "${2:0:1}" == "-" ]]; then
+              print_usage "$SCRIPT_CMD" "Missing value for the --solr-disk-gb parameter!"
+              exit 1
+            fi
+            SOLR_DISK_GB=$2
+            shift 2
+        ;;
+        --node-pool)
+            if [[ -z "$2" || "${2:0:1}" == "-" ]]; then
+              print_usage "$SCRIPT_CMD" "Missing value for the --node-pool parameter!"
+              exit 1
+            fi
+            NODE_POOL="$2"
+            shift 2
+        ;;
+        -help|-usage|--help|--usage)
+            print_usage "$SCRIPT_CMD"
+            exit 0
+        ;;
+        --)
+            shift
+            break
+        ;;
+        *)
+            if [ "$1" != "" ]; then
+              print_usage "$SCRIPT_CMD" "Unrecognized or misplaced argument: $1!"
+              exit 1
+            else
+              break # out-of-args, stop looping
+            fi
+        ;;
+    esac
+  done
+fi
+
+if [ "$CLUSTER_NAME" == "" ]; then
+  print_usage "$SCRIPT_CMD" "Please provide the K8s cluster name using: -c <cluster>"
+  exit 1
+fi
+
+if [ "$PROVIDER" != "gke" ]; then
```

Comment:
This change is required in case customize_fusion_values.sh is executed before setup_f5_eks.sh
In case we executed withouit this change,  pods will fail with:   Warning  FailedScheduling  2m50s (x28 over 19m)  default-scheduler  0/3 nodes are available: 3 node(s) didn't match node selector.